FluentConstraints/FluentConstraintSet.swift

```diff
@@ -17,36 +17,36 @@ public class FluentConstraintSet {
         self.firstView = view
     }
 
-    public func build() -> [NSLayoutConstraint] {
+    open func build() -> [NSLayoutConstraint] {
         return constraints.map { $0.build() }
     }
 
-    public func activate() -> [NSLayoutConstraint] {
+    open func activate() -> [NSLayoutConstraint] {
         let constraints = build()
-        NSLayoutConstraint.activateConstraints(constraints)
+        NSLayoutConstraint.activate(constraints)
         return constraints
     }
 
     // MARK: relationship to view
 
-    public var inSuperview: FluentConstraintSet {
+    open var inSuperview: FluentConstraintSet {
         precondition(self.firstView.superview != nil, "View does not have a superview")
         self.constraints.forEach { $0.secondView = self.firstView.superview! }
         return self
     }
 
-    public func onView(view: UIView) -> FluentConstraintSet {
+    open func onView(_ view: UIView) -> FluentConstraintSet {
```

Comment:
Might rename this one to `on(view:)` to better match Swift 3's avoidance of repeating type information in function names.